packages/field-base/src/input-constraints-mixin.js

```diff
@@ -70,6 +70,30 @@ const InputConstraintsMixinImplementation = (superclass) =>
       }
     }
 
+    /**
+     * Override an event listener inherited from `InputMixin`
+     * to capture native `change` event and make sure that
+     * a new one is dispatched after validation runs.
+     * @param {Event} event
+     * @protected
+     * @override
+     */
+    _onChange(event) {
+      event.stopImmediatePropagation();
```

Comment:
nit: This probably wouldn't need to be immediate.

Reply:
Updated to use `stopPropagation`.